### DISCUSSION TITLE
From a list of titles create a new note for each one

### DISCUSSION THREAD

**Discussion by leebarkerlbrkr** [SilentVoid13/Templater#1408] [Help]
Hi, I hope you can guide me with this.
Id like to select all items in a list and to use a templater hotkey to create a new note for each one.

At the moment Im making a new note one by one and I have hundreds of tutorials in the list.
i know this is the "great pain" of setting an Obisidian repository up but I was hoping I could find some automation to help.
Thanks to anyone who can help
Lee

**Comment by Zachatoo** [comment]
Here's an example of taking each line in your selection and creating a new note from that selection. If it isn't exactly what you want, it should be a good starting point.

```js
<%*
const lines = tp.file.selection().split("\n");
for (let line of lines) {
  await tp.file.create_new("[[Template to create new note with]]", line);
}
tR += tp.file.selection();
-%>
```

**Reply by leebarkerlbrkr** [reply]
Thank you so so much . This is so kind of you to post for me.
I'll try this out tomorrow and let you know how I get on. Either way I really appreciate it.
